clang/unittests/ASTMatchers/ASTMatchersNodeTest.cpp

```diff
@@ -1912,6 +1912,21 @@ TEST_P(ASTMatchersTest, DeducedTemplateSpecializationType) {
               deducedTemplateSpecializationType()));
 }
 
+TEST_P(ASTMatchersTest, DependentNameType) {
+  if (!GetParam().isCXX()) {
+    // FIXME: Add a test for `dependentNameType()` that does not depend on C++.
```

Comment:
Similar to `DependentScopeDeclRefExpr`, `DependentNameType` only exists in C++, so this FIXME comment can be removed.

Reply:
Done